go-controller/pkg/util/ovs_unit_test.go

```diff
@@ -488,6 +489,60 @@ func TestRunOVSOfctl(t *testing.T) {
 	}
 }
 
+func TestRunOVSDpctl(t *testing.T) {
+	mockKexecIface := new(mock_k8s_io_utils_exec.Interface)
+	mockExecRunner := new(mocks.ExecRunner)
+	mockCmd := new(mock_k8s_io_utils_exec.Cmd)
+	// below is defined in ovs.go
+	runCmdExecRunner = mockExecRunner
+	// note runner is defined in ovs.go file
+	runner = &execHelper{exec: mockKexecIface}
+	tests := []struct {
+		desc                    string
+		expectedErr             error
+		onRetArgsExecUtilsIface *onCallReturnArgs
+		onRetArgsKexecIface     *onCallReturnArgs
+	}{
+		{
+			desc:                    "negative: run `ovs-dpctl` command",
+			expectedErr:             fmt.Errorf("executable file not found in $PATH"),
+			onRetArgsExecUtilsIface: &onCallReturnArgs{"RunCmd", []string{"*mocks.Cmd", "string", "[]string"}, []interface{}{nil, nil, fmt.Errorf("executable file not found in $PATH")}},
```

Comment:
L508: Calling `RunOVSDpctl` will never return `"executable file not found in $PATH"`. That error will be returned by SetExec() which ensures that the executable is at the right location. So, by the time we call RunOVSDpctl() we already know that the container has ovs-dpctl executable.

I know these are mocks, but for correctness return any error .. say "failed to execute ovs-dpctl command". Similarly, for all the other commands in this file.

Reply:
@girishmg yes, you are right in your observation